jest.config.js

```diff
@@ -23,7 +23,16 @@ module.exports = {
         '^.+\\.module\\.(css|sass|scss)$',
         '<rootDir>/node_modules/(?!lodash-es|@stencil)',
       ],
-      testPathIgnorePatterns: ['/node_modules/'],
+      testPathIgnorePatterns: [
+        '/node_modules/',
+        '/apps/pci-vouchers/',
+        '/apps/pci-ssh-keys/',
+        '/apps/pci-users/',
+        '/apps/pci-public-ip',
+        '/apps/pci-gateway/',
+        '/apps/pci-private-network',
+        '/apps/key-management-service',
```

Comment:
Why you ignore theses micro apps here, we already fix this in this PR https://github.com/ovh/manager/pull/12279/files